config/webpack.config.js

```diff
@@ -11,12 +11,10 @@ const dir_build = path.resolve(__dirname, basePath, 'build');
 
 module.exports = {
     entry: {
-        main: path.resolve(dir_js, 'main.js'),
-        vendor: [
-            'babel-polyfill',
```

Comment:
Removing this should be carefully double checked in order to avoid undesired side-effects